packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -730,6 +730,29 @@ Fetches a user from your Okta organization
 - [Get User with Login](#get-user-with-login)
 - [Get User with Login Shortname](#get-user-with-login-shortname)
 
+##### Content-Type Parameters
+
+All variants of this API support a performance optimization.  Complex DelAuth configurations may degrade performance when fetching specific parts of the response.  Passing an optional Content-Type parameter can omit these parts, bypassing the bottleneck.
+
+| Parameter     | Description                                                                                                                           |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
+| okta-response | A comma-separated list of omit options (optionally surrounded in quotes) each specifying a specific part of the resonse to be omitted |
```

Comment:
`resonse` --> `response`
also, can we come up with a less redundant phrasing than "specifying a specific part"?